File: get_feature_329.js
const fs = require('fs');

// Read the features database
const data = JSON.parse(fs.readFileSync('assistant.db', 'utf-8'));

// Find feature 329
const feature = data.features.find(f => f.id === 329);
if (feature) {
    console.log(JSON.stringify(feature, null, 2));
} else {
    console.log('Feature 329 not found');
}

File: get_feature_329.py
import json

# Read the features database
with open('assistant.db', 'r', encoding='utf-8') as f:
    data = json.load(f)

# Find feature 329
for feature in data.get('features', []):
    if feature.get('id') == 329:
        print(json.dumps(feature, indent=2))
        break
